### PR TITLE
added whitespace and oneOrMore functions

### PR DESCRIPTION
**Pull request by ptrstpp950**
To make code more readable I added `whitespace` shortcut.

Second thing is to add `oneOrMore` function because it is usefull in many regexp. I added full explanation on my blog http://stapp.space/clean-code-regular-expression/

For both methods I included tests :)

### CODE REVIEW

**Comment by softwarespot** [comment]
Based on your well written post, this makes perfect, logical sense.

**Comment by ptrstpp950** [comment]
I'm glad that you like it

**Comment by ionutvmi** [comment]
+1
Looks good to me.

**Comment by ionutvmi** [comment]
@ptrstpp950 can you fix the conflicts so we can merge it please ?

**Comment by ptrstpp950** [comment]
@ionutvmi done

**Comment by shakuzen** [comment]
Should these two methods be added to the [implementation Wiki](https://github.com/VerbalExpressions/implementation/wiki/List-of-methods-to-implement) as well?

File: VerbalExpressions.js [hunk 1]
```diff
@@ -310,6 +316,13 @@
             return (this);
         },
 
+        // Repeats the previous at least ones
```
Comment (ionutvmi) [inline VerbalExpressions.js]: Just noticed we have a typo here.

**Comment by ionutvmi** [comment]
@shakuzen they were added.

File: VerbalExpressions.js (hunk 1 above)
Comment (ptrstpp950) [inline VerbalExpressions.js]: :+1:

**Comment by ptrstpp950** [comment]
@ionutvmi next time I will fix also wiki

**Comment by goldingdamien** [comment]
What about this wiki?
https://github.com/VerbalExpressions/JSVerbalExpressions/wiki

**Comment by shreyasminocha** [comment]
@goldingdamien The wiki is incomplete. Support #168 for complete documentation in `.md` + GH pages format.

**Pull request merged**